pkgs/by-name/j2/j2me/package.nix

```
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ant
+, jdk
+, stripJavaArchivesHook
+, SDL2
+, freeimage
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "j2me";
```

```suggestion
  pname = "freej2me";
```

These seem to be more j2me emulators than just freej2me.